src/lib/serialize-integration-markdown.ts

```diff
@@ -0,0 +1,22 @@
+import rehypeSurfaceCodeNewlines from '@hashicorp/platform-code-highlighting/rehype-surface-code-newlines'
+import { paragraphCustomAlerts, typography } from '@hashicorp/remark-plugins'
+import rehypePrism from '@mapbox/rehype-prism'
+import { MDXRemoteSerializeResult } from 'next-mdx-remote'
+import { serialize } from 'next-mdx-remote/serialize'
+
+// TODO: export types from `next-mdx-remote` v3
+const SERIALIZE_OPTIONS: Parameters<typeof serialize>[1] = {
+	mdxOptions: {
+		remarkPlugins: [paragraphCustomAlerts, typography],
+		rehypePlugins: [
+			[rehypePrism, { ignoreMissing: true }],
+			rehypeSurfaceCodeNewlines,
+		],
+	},
+}
+
+export default async function serializeIntegrationMarkdown(
```

Comment:
```suggestion
nice, thank you for lifting this up into its own file :D 
```

Reply:
Hmm I'm not sure about this change 😆

Reply:
> _sir, that's not valid TypeScript_

Reply:
We are now prompt developers, plug that into chatGPT and see what it does